Frontend/src/views/Start.vue

```diff
@@ -1,8 +1,15 @@
 <template>
```

Comment:
Eigentlich gehört das hier nicht rein. Ich würde die Änderungen an der Start.vue in einem anderen Branch hinzufügen und hier herausnehmen